gregBorchers/HW6Problem1.js

```diff
@@ -0,0 +1,91 @@
+
```

Comment:
Looks good! I've got a few minor comments in here, but other than those it's solid stuff. Nice work!